students/17th/mikyung/styles/login.css

```
@@ -0,0 +1,146 @@
+body {
+  background-color: #fafafa;
+}
+
+h1 {
+  text-align: center;
+  font-family: "Cookie", cursive;
+  font-size: 50px;
+}
+
+.first_div {
+  width: 350px;
+  height: 380px;
+  margin-top: 44px;
+  margin-right: auto;
+  margin-left: auto;
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  text-align: center;
+  flex-wrap: wrap;
+  background-color: #fff;
+  border: 1px solid #dbdbdb;
+}
+
+.input_login {
+  width: 258px;
+  height: 36px;
+  margin: 3px auto;
+  padding: 0 10px;
+  display: block;
+  background-color: #fafafa;
+  border: 1px solid #dbdbdb;
+  border-radius: 3px;
+  font-size: 14px;
+}
+
+.input_login::placeholder {
+  font-size: 12px;
+  color: #8e8e8e;
+}
+
+.input_login:focus {
+  border: 1px solid #c0c0c0;
+  outline: none;
+}
+#btn_login {
+  width: 280px;
+  height: 30px;
+  margin: 13px;
+  background-color: #b2dffc;
+  border: none;
+  border-radius: 3px;
+  color: white;
+  font-size: 14px;
+  font-weight: 600;
+}
+
+.or_box {
+  border: 1px solid #dbdbdb;
+  border-right: none;
+  border-left: none;
+  border-top: none;
+  width: 280px;
+  position: relative;
+  margin: 8px auto;
+}
+span.or_under {
```

```suggestion
}

span.or_under {
```
새로운 셀렉터 끼리는 가독성을 위해 한줄 씩 띄워주세요!